src/panels/lovelace/cards/hui-glance-card.ts

```diff
@@ -235,6 +235,7 @@ export class HuiGlanceCard extends LitElement implements LovelaceCard {
                 .stateObj=${stateObj}
                 .overrideIcon=${entityConf.icon}
                 .overrideImage=${entityConf.image}
+                .stateColor="true"
```

Comment:
add option?

Reply:
Let's not start with an option. Like, who wants a glance card without being able to see what's going on. 

However, this is not how you define boolean properties.
```suggestion
                statecolor
```